packages/client/src/components/CommentCard.js

```diff
@@ -1,13 +1,15 @@
 import React, { useContext, useState } from 'react';
 import { ConfigContext } from '../context';
-import timeAgo from '../utils/timeAgo';
+import { timeAgo } from '../utils';
 import CommentBox from './CommentBox';
 
 export default function CommentCard({ comment, boxConfig, rootId, onSubmit }) {
   const [reply, setReply] = useState(null);
 
   const ctx = useContext(ConfigContext);
 
+  // This is remained only because of existing comments
+  // Links is now handled when inputing
```

Comment:
This function can be removed in the future. But it will effect existing comments.